client/question/services/ConversationManagerServiceTestUtils.js

```diff
@@ -79,38 +70,30 @@ tie.factory('ExpectedFeedbackObjectFactory', [function() {
    */
   ExpectedFeedback.prototype.verifyFeedback = function(submissionResult) {
     var errorMessages = [];
+    var observedFeedback = submissionResult.getFeedback();
+    var observedFeedbackParagraphObject = observedFeedback.getParagraphs();
```

Comment:
Variable name sounds singular, but getParagraphs() presumably returns a list. Consider renaming to observedFeedbackParagraphs.

Reply:
Actually... it's an array of feedback paragraph objects; so, it seems "Object" should be in the name. Would observedFeedbackParagraphsObject work?

Reply:
Typically, we omit the "object" when referring to domain objects by name, so I think observedFeedbackParagraphs is better here and more consistent with the rest of the codebase. (Otherwise you'll also need to do getParagraphObjects() etc.)

(Note also that it would be Object**s** in that case.)

Reply:
ACK Fixed in upcoming commit.